sdk/txbuild/close.go

```diff
@@ -27,8 +27,9 @@ func Close(p CloseParams) (*txnbuild.Transaction, error) {
 			AccountID: p.InitiatorEscrow.Address(),
 			Sequence:  startSequenceOfIteration(p.StartSequence, p.IterationNumber) + 1, // Close is the second transaction in an iteration's transaction set.
 		},
-		BaseFee:              0,
-		Timebounds:           txnbuild.NewTimeout(300),
+		BaseFee: 0,
+		// TODO - Timebounds needs to be explicit
+		Timebounds:           txnbuild.NewInfiniteTimeout(),
```

Comment:
Good catch. These should be infinite since a channel is designed to be open indefinitely.
```suggestion
		BaseFee:              0,
		Timebounds:           txnbuild.NewInfiniteTimeout(),
```